recipes/meta/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "meta" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "a35dd5bc3ccffaaf83aa488233467ae85dc5374aea3068c34a6dc85ea29e27a9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bagari/meta/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage('meta', max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - cmake
+    - make  # [not win]
+
+  host:
+    - python >=3.9,<3.12
+    - pip
+    - setuptools
+    - libboost-python-devel
+    - itk >=5.0.0,<=5.3.0
+    - libitk >=5.0.0,<=5.3.0
+    - libitk-devel >=5.0.0,<=5.3.0
+    - mesalib
```

Comment:
you likely want something like https://conda-forge.org/docs/maintainer/knowledge_base/#libgl